### PR TITLE
Update ru_ru.json

### PR DESCRIPTION
**Pull request by BardinTheDwarf**

### CODE REVIEW

File: src/main/resources/assets/jeresources/lang/ru_ru.json [hunk 1]
```diff
@@ -1,129 +1,164 @@
...
+    "jer.config.enchantsBlacklist.description": "Убирает зачарования из JEI-просмотра зачарований",
+    "jer.config.diyData.title": "Использовать данные из модов",
+    "jer.config.diyData.description": "Встроенный код совместимости не будет загружаться, и json-ы будут загружены (требуется перезапуск)",
+    "jer.config.dimensionsBlacklist.title": "Черный список измерений",
+    "jer.config.dimensionsBlacklist.description": "Убирает измерения из профилирующего сканирования JEI",
+	"jer.config.showDevData.title": "Показать дополнительную информацию",
+	"jer.config.showDevData.description": "В основном для создателей сборок. Функции включают в себя, но не ограничиваются средн. блоков на чанк, ID существ, ...",
+	"jer.config.hiddenTabs.title": "Скрыть вкладки JER",
+	"_comment": "# do not translate strings between quotes as they are used in code",
+	"jer.config.hiddenTabs.description": "Вкладки добавленные JER 'dungeon', 'enchantment', 'mob', 'villager', worldgen' и 'plant'",
```
Comment (CrafterKolyan) [inline src/main/resources/assets/jeresources/lang/ru_ru.json:16]: Seems that a quote was missed.
```suggestion
	"jer.config.hiddenTabs.description": "Вкладки добавленные JER 'dungeon', 'enchantment', 'mob', 'villager', 'worldgen' и 'plant'",
```

File: src/main/resources/assets/jeresources/lang/ru_ru.json [hunk 2]
```diff
@@ -1,129 +1,164 @@
...
 
-"_comment": "#Default dimensions",
-"jer.dim.minecraft:overworld": "Верхний мир",
-"jer.dim.minecraft:the_nether": "Незер",
-"jer.dim.minecraft:the_end": "Эндер",
+    "_comment": "#Default dimensions",
+	"jer.dim.all": "Все",
+    "jer.dim.minecraft:overworld": "Верхний мир",
+    "jer.dim.minecraft:the_nether": "Незер",
+    "jer.dim.minecraft:the_end": "Эндер",
```
Comment (CrafterKolyan) [inline src/main/resources/assets/jeresources/lang/ru_ru.json:22]: Here is a correct russian translation.
```suggestion
    "jer.dim.minecraft:the_end": "Край",
```
Consider: https://minecraft-ru.gamepedia.com/%D0%9A%D1%80%D0%B0%D0%B9

File: src/main/resources/assets/jeresources/lang/ru_ru.json [hunk 3]
```diff
@@ -1,129 +1,164 @@
...
+	"jer.config.hiddenTabs.description": "Вкладки добавленные JER 'dungeon', 'enchantment', 'mob', 'villager', worldgen' и 'plant'",
 
-"_comment": "#Default dimensions",
-"jer.dim.minecraft:overworld": "Верхний мир",
-"jer.dim.minecraft:the_nether": "Незер",
-"jer.dim.minecraft:the_end": "Эндер",
+    "_comment": "#Default dimensions",
+	"jer.dim.all": "Все",
+    "jer.dim.minecraft:overworld": "Верхний мир",
+    "jer.dim.minecraft:the_nether": "Незер",
```
Comment (CrafterKolyan) [inline src/main/resources/assets/jeresources/lang/ru_ru.json:21]: This is one of the most common term for the Nether in Russian
```suggestion
    "jer.dim.minecraft:the_nether": "Ад",
```
https://minecraft-ru.gamepedia.com/%D0%9D%D0%B8%D0%B6%D0%BD%D0%B8%D0%B9_%D0%BC%D0%B8%D1%80

File: src/main/resources/assets/jeresources/lang/ru_ru.json [hunk 4]
```diff
@@ -1,129 +1,164 @@
...
+    "_comment": "#Mob View",
+    "jer.mob.title": "Выпадение из мобов",
+    "jer.mob.spawn": "Появляется в биоме:",
+    "jer.mob.biome": "Появляется в биомах: наведите курсор сюда",
+    "jer.mob.exp": "Даёт опыта",
 
-"_comment": "#Villager View",
-"jer.villager.title": "Торговля с крестьянином",
+    "_comment": "#Villager View",
+    "jer.villager.title": "Торговля с крестьянином",
```
Comment (CrafterKolyan) [inline src/main/resources/assets/jeresources/lang/ru_ru.json:103]:
```suggestion
    "jer.villager.title": "Торговля с жителем",
```
https://minecraft-ru.gamepedia.com/%D0%94%D0%B5%D1%80%D0%B5%D0%B2%D0%B5%D0%BD%D1%81%D0%BA%D0%B8%D0%B9_%D0%B6%D0%B8%D1%82%D0%B5%D0%BB%D1%8C

**Review by CrafterKolyan — CHANGES_REQUESTED** [review]

**Comment by way2muchnoise** [comment]
@BardinTheDwarf please review the suggested changes

**Pull request merged**